isaric/parsers/isaric-ecmo.toml

```diff
@@ -550,6 +550,20 @@
   description = "Admitted to ICU?"
   apply = { function = "isNotNull" }
 
+[subject.vaccinated_covid19]
+  field = "vacc_cov19_cmyn"
+  values = { 1 = true, 0 = false }
+
+[subject.vaccinated_covid19_dates]
+  combinedType = "list"
+  excludeWhen = "none"
+  fields = [
+    { field = "vacc_cov19_1_cmdat" },
+    { field = "vacc_cov19_2_cmdat" },
+    { field = "vacc_cov19_3_cmdat" },
+    { field = "vacc_cov19_cmdat" },
```

Comment:
vacc_cov19_cmdat was on an earlier version of the CRF, collecting date of most recent dose (associated with vacc_cov19_recent_cmtrt indicating the type of most recent vaccination). The other date variables are dates of individual doses collected on a more recent version of the CRF. There are instances where the only date we have is from the earlier version, maybe a context should be added to this to indicate most recent dose.